components/Explorer/Explorer.jsx

```diff
@@ -5,12 +5,9 @@ require('brace/theme/monokai');
 import fetch from 'isomorphic-fetch';
 import { HOST_URL } from '../../yasp.config';
 import RaisedButton from 'material-ui/RaisedButton';
+import TextField from 'material-ui/TextField';
```

Comment:
Not sure, but Spinner, Popover, etc. should be there and this diff shows them being removed.